bigchaindb/common/vote.py

```diff
@@ -6,16 +6,16 @@
 from bigchaindb.common.schema import (_validate_schema,
                                       TX_SCHEMA_COMMON,
                                       TX_SCHEMA_TRANSFER,
-                                      TX_SCHEMA_VALIDATOR_ELECTION_VOTE)
+                                      TX_SCHEMA_VOTE)
 
 
-class ValidatorElectionVote(Transaction):
+class Vote(Transaction):
 
-    VALIDATOR_ELECTION_VOTE = 'VALIDATOR_ELECTION_VOTE'
+    VOTE = 'VOTE'
     # NOTE: This class inherits TRANSFER txn type. The `TRANSFER` property is
     # overriden to re-use methods from parent class
-    TRANSFER = VALIDATOR_ELECTION_VOTE
-    ALLOWED_OPERATIONS = (VALIDATOR_ELECTION_VOTE,)
+    TRANSFER = VOTE
+    ALLOWED_OPERATIONS = (VOTE,)
```

Comment:
`OPERATION = VOTE` should also be specified.

Reply:
done

Reply:
I don't see `OPERATION = VOTE` specified anywhere